### PR TITLE
New template: Doppio choking

### PR DESCRIPTION
**Pull request by caiorulli-quintoandar**
Hi! I tried adding a new meme template.
Really liked the idea for the repo! Hope I'll be able to contribute further! :+1: 

- [Meme origin](https://knowyourmeme.com/memes/choking-doppio)
- [Original template](https://www.reddit.com/r/ShitPostCrusaders/comments/dad8h9/doppio_choking_meme_template/)
- Example: 
doppiomeme

### CODE REVIEW

**Comment by fakefred** [comment]
~~JOOOJOOOOOOOOO~~ Very cool, thanks! I will test this and merge, like, tomorrow cos I'm busy

File: memethesis/meme/res/template/doppio/format.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  panels:
+    choking:
+      description: Doppio choking
+      image: doppio/choking.jpg
+      textbox: [600, 60, 425, 330]
+    ok:
+      description: Doppio ok
+      image: doppio/ok.jpg
+      textbox: [600, 60, 425, 330]
+    chokingharder:
```
Comment (fakefred) [inline memethesis/meme/res/template/doppio/format.yml]: Personally I prefer short tags. Maybe `harder`?

**Review by fakefred — COMMENTED** [review]
CLI commands will look nicer if the choking harder flag is shortened to `harder`.

File: memethesis/meme/res/template/doppio/format.yml (hunk 1 above)
Comment (caiorulli-quintoandar) [reply memethesis/meme/res/template/doppio/format.yml]: Nice! Makes total sense! Will do

**Comment by fakefred** [comment]
Perfect! Merged.

**Pull request merged**